app/javascript/onboarding/components/Navigation.jsx

```diff
@@ -1,16 +1,36 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
 
-const Navigation = ({ next, prev, hideNext, hidePrev }) => (
-  <nav className="onboarding-navigation">
-    <div className="navigation-content">
+const Navigation = ({
+  next,
+  prev,
+  hideNext,
+  hidePrev,
+  disabled,
+  className,
+}) => (
+  <nav
+    className={`onboarding-navigation${
+      className && className.length > 0 ? ` ${className}` : ''
+    }`}
+  >
+    <div
+      className={`navigation-content${
+        className && className.length > 0 ? ` ${className}` : ''
+      }`}
+    >
       {!hidePrev && (
         <button onClick={prev} className="back-button" type="button">
           Back
         </button>
       )}
       {!hideNext && (
-        <button onClick={next} className="next-button" type="button">
+        <button
```

Comment:
Consider using the [design system <Button />](https://devto-storybook.now.sh/?path=/story/components-buttons--description) component anywhere `<button />` is currently being used.

Reply:
If it's okay with you, I'm thinking we can do this in a separate PR as well. There are a lot of slides in the onboarding flow that could be updated with the new design system button...and I like the idea of just updating the variables, buttons, etc in one fell swoop. I can add a card to [our project board](https://github.com/thepracticaldev/dev.to/projects/11#column-8471698) for this so that we make sure it gets done in #cooldown !